### DISCUSSION TITLE
What subprotocol is used to send bold fonts in the RDP protocol?

### DISCUSSION THREAD

**Discussion by Mr-LZG** [FreeRDP/FreeRDP#10384] [General]
What subprotocol is used to send bold fonts in the RDP protocol? Why is the bold font not sent using the glyph protocol? The above situation occurs when using freerdp-proxy.

**Comment by akallabeth** [comment]
@Mr-LZG because windows XP is no longer and direct GDI mapping no longer a thing.
everything else is using https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpegfx/da5c75f9-cd99-450c-98c4-014a496942b0 or some bitmap transfer as fallback.
